documentation/c10-classdesign.sil

```diff
@@ -243,17 +241,17 @@ Once again this is done in an object-oriented way,
 with derived classes overriding their superclass’ methods where necessary.
 
 Some packages may provide functions that need to be run as part of the class output routines.
-One way packages can accomplish this is registering hook functions that get run at known locations in the provided classes.
+One way packages can accomplish this is by registering hook functions that get run at known locations in the provided classes.
 Class authors may also provide their own hook locations for packages, or run any set of registered hooks in their own outputs.
 The other way is to export functions that get added to the class itself and can be run manually from the class.
 
-For examples check out the \autodoc:package{tableofcontents} package for the hooks it sets, but also the \autodoc:command{\tocentry} command it registered that gets called manually in the \code{book} class.
+For examples check out the \code{tableofcontents} package for the hooks it sets, but also the \autodoc:command{\tocentry} command it registers that gets called manually in the \code{book} class.
```

Comment:
Why `\autodoc:package{tableofcontents} package` > `\code{tableofcontents}` here? (The idea being that package names are typeset as `\autodoc:package{}` rather than code, for possible cross-reference/indexing later)

Reply:
Yes, I think this was my mistake. You’re right, it should be \autodoc:package{}.

Reply:
Touched up.